pkg/cmd/test.go

```diff
@@ -586,6 +578,8 @@ func runScript(scriptFile, desc, namespace, baseDir string) error {
 
 	command.Dir = baseDir
 
+	command.Stderr = os.Stdout
+
 	if out, err := command.Output(); err == nil {
```

Comment:
```suggestion
	command.Stdout = os.Stdout
	command.Stderr = os.Stderr
	
	if out, err := command.Run(); err == nil {
```

Reply:
Maybe this configuration works better, so the execution is all written live to the logs

Reply:
Mmh.. `command.Run()` returns only the error, but you've got what I mean..

Reply:
updated.